### PR TITLE
Fix/pn 7763

### PR DESCRIPTION
**Pull request by leleOFA**
## Short description
delete filter with cancellation in progress
## List of changes proposed in this pull request
- delete filter with cancellation in progress
- delete fetch paymentinfo when notification is cancelled
- fix test with fetch payment info

## How to test
in dashbaord pa not view filter with status 'cancellation in progress'

### CODE REVIEW

File: packages/pn-personafisica-webapp/src/pages/__test__/NotificationDetail.page.test.tsx [hunk 1]
```diff
@@ -154,10 +154,10 @@ describe('NotificationDetail Page', () => {
 
   // Questo test fallisce per un bug nel dettaglio notifica. La paymentInfo viene chiamata anche se la notifica è in stato annullata
```
Comment (AndreaCimini90) [inline packages/pn-personafisica-webapp/src/pages/__test__/NotificationDetail.page.test.tsx:155]: remove the comments

File: packages/pn-personafisica-webapp/src/component/Notifications/__test__/NotificationPayment.test.tsx [hunk 2]
```diff
@@ -509,9 +509,7 @@ describe('NotificationPayment component', () => {
     expect(button).not.toBeInTheDocument();
   });
 
-  // Questo test fallisce per un bug nel dettaglio notifica. La paymentInfo viene chiamata anche se la notifica è in stato annullata
-  // Rimuovere il commento e lo skip del test una volta sistemato il bug
-  it.skip('renders payment when notification is CANCELLED', async () => {
```
Comment (AndreaCimini90) [inline packages/pn-personafisica-webapp/src/component/Notifications/__test__/NotificationPayment.test.tsx:514]: Do the things under COSE DA FARE

**Review by AndreaCimini90 — CHANGES_REQUESTED** [review]

File: packages/pn-personafisica-webapp/src/pages/__test__/NotificationDetail.page.test.tsx (hunk 1 above)
Comment (leleOFA) [reply packages/pn-personafisica-webapp/src/pages/__test__/NotificationDetail.page.test.tsx:155]: done

File: packages/pn-personafisica-webapp/src/component/Notifications/__test__/NotificationPayment.test.tsx (hunk 2 above)
Comment (leleOFA) [reply packages/pn-personafisica-webapp/src/component/Notifications/__test__/NotificationPayment.test.tsx:514]: do it

**Pull request merged**